modifiers/caliper-mpi/configs/modifier.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+variables:
+  modifier_package_name: 'caliper-mpi'
+  modifier_spack_variant: '+caliper'
+
+modifiers:
+  - name: caliper-mpi
+    mode: mpi
+
+spack:
+  packages:
+    caliper:
+      spack_spec: caliper+adiak+mpi~libunwind~libdw~papi
```

Comment:
Currently we'll need `caliper@master` for the communication region annotation macros